### DISCUSSION TITLE
kernel: Bluetooth: hci0: corrupted SCO packet

### DISCUSSION THREAD

**Discussion by DittelHome** [linuxmint/discussions#662] [Questions]
Hello community,
I'm trying to get my Bluetooth headset "New Bee LC-B45" up and running. Pairing worked perfectly. Sound playback worked too. According to the Sound Manager, the microphone only works if I select "Headset Head Unit (HSP/HFP Codec CVSD)" as the audio profile. Then the input level is at least displayed. A recording with "arecord --duration=5 test.wav" is disappointing, all you hear is scratching. 

During the tests, I get hundred's of this error messages:
**kernel: Bluetooth: hci0: corrupted SCO packet**

I use:
Distro: Linux Mint 22 Wilma base: Ubuntu 24.04 noble
Kernel: 6.8.0-47-generic
Bluetooth:  Intel Bluetooth wireless interface driver: btusb v: 0.8 type: USB rev: 2.0
Laptop System: LENOVO product: 80KX v: Lenovo E31-70 
Fresh install of mint

Where can I address the problem?
or
What can i do, to fix the problem ..?

regards Klaus

**Comment by DittelHome** [comment]
To create an issue I have to assign the problem to a repository. But I can't find anything regarding the kernel.
Blueman is only the Bluetooth Manager.
Where can I address the problem ?
Sorry for my stupid questions.

**Reply by Aura-Monster69** [reply]
Did you manage to fix the problem? I am having the same issue.

**Reply by DittelHome** [reply]
No, I’m now using a Jabra headset, including the corresponding Bluetooth dongle. It works perfectly with that.
